Bottom: ef9e079da327d1ac5af5d3bc52813ba00a4f18dd
Top:    44f0d2ffebd524b0d36e3887ca954fef38a38dc1
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 01:18:11 +0000

Refresh of well-on-the-way-to-completely

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 6e2bf29..58f3c6f 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -305,6 +305,7 @@ object3d_from_board_outline (void)
             }
 
           get_contour_coord_n_in_step_mm (ct, offset_in_ct, &x1, &y1);
+
           vertices[i]           = make_vertex3d (x1, y1, -COORD_TO_STEP_Z (PCB, HACK_BOARD_THICKNESS)); /* Bottom */
           vertices[npoints + i] = make_vertex3d (x1, y1, 0);                                            /* Top */
